messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -621,20 +621,6 @@ class MessagingTest : BaseMessagingTest() {
         }
     }
 
-    @Test
-    fun testMyself() {
-        testInCoroutine {
-            newDB.use { dogDB ->
-                newMessaging(dogDB, "dog").with { dog ->
-                    val dogId = dog.myId.id
-                    dog.addOrUpdateDirectContact(dogId, "Note to self")
```

Comment:
I'm sorry if I wasn't clear, I think this test is useful, I just mean that you don't need this line (line 630).

Reply:
My bad, I understood that the whole test was unnecessary 😰